Storage/src/Connection/RetryTrait.php

```diff
@@ -105,6 +108,7 @@ public function getRestRetryFunction($resource, $method, $args)
         $isOpIdempotent = in_array($methodName, $this->idempotentOps);
         $preconditionNeeded = array_key_exists($methodName, $this->condIdempotentOps);
         $preconditionSupplied = $this->isPreConditionSupplied($methodName, $args);
+        $retryStrategy = isset($args['retryStrategy']) ? $args['retryStrategy'] : null;
```

Comment:
Perhaps we can use `self::$RETRY_STRATEGY_IDEMPOTENT` instead of null as a default?
That way we can even remove it from the default case below.

Reply:
+1

Reply:
Okay, implemented in this commit [a47e376](https://github.com/googleapis/google-cloud-php/pull/5731/commits/a47e3761927e63746dafff9048bc0c8b17b74bb0)